src/cache/cache-list.js

```diff
@@ -1,39 +0,0 @@
-const AWS = require('aws-sdk');
-const config = require('../../config');
-
-const getRedisEndpoints = async () => {
-  const ec = new AWS.ElastiCache({
-    region: config.awsRegion,
-  });
-
-  var params = {
-    Marker: 'STRING_VALUE',
-    MaxRecords: 'NUMBER_VALUE',
-    ReplicationGroupId: 'STRING_VALUE'
-  };
-
-  const cluster = await ec.describeReplicationGroups({
-    ReplicationGroupId: `biomage-redis-${clusterEnv}`
-  }).promise();
-
-
-  // biomage - redis - production
-}
-
-
-module.exports = {
-  'eu-west-1': {
```

Comment:
I wonder how did that ever work if it is saying eu-west-1 and we used to be in eu-west-2

Reply:
I changed that in a previous commit I'm pretty sure when I was migrating to GitHub. It says `eu-west-2` on GiLab, see [here](https://gitlab.com/biomage/api/-/blob/master/src/cache/cache-list.js).